code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -74,7 +74,7 @@
 	name = "warden's shotgun"
 	desc = "A heavily modified Hephaestus Industries KS-40. This version bears multiple after-market mods, including a laser sight to help compensate for its shortened stock. 'Property of the Warden' has been etched into the side of the reciever. Uses 12g rounds."
 	icon_state = "shotgun_w"
-	slot_flags = SLOT_BELT|SLOT_HOLSTER|SLOT_BACK
```

Comment:
Did you mean to do this? Cause you have this PR: https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/4807/files